test/cpp/jit/tests.h

```diff
@@ -2067,6 +2068,17 @@ void testAliasAnalysis() {
         usesB->node(), mutatesAliasOfB->node()));
   }
 }
+
+void testEvalModeForLoadedModule() {
+  std::string module_path = "dropout_model.pt";
+  std::shared_ptr<torch::jit::script::Module> module = torch::jit::load(module_path);
+  // Test eval mode
```

Comment:
I think the comments here don't add too much

Reply:
removed